tests/issues/issue-417/unformatted.pug

```diff
@@ -0,0 +1,3 @@
+p Aliquam pulvinar enim eu enim finibus euismod. 
```

Comment:
Ah I see, reminder for my future me: handle the whitespace of non-trimmed version of the string (text-token)